### PR TITLE
✨ JS-609: Adding an Event Listener

### PR DESCRIPTION
**Pull request by ct-aa**
# Related Tickets
- [JS-609](https://codeandtheory.atlassian.net/browse/JS-609)

# Description
Added the **Adding an Event Listener** challenge solutions.

# Validation
* Open the **Adding an Event Listener** folder.
* Open the app.js.
* .An event listener should be added to the variable button that adds a background color styling to the element assigned to the variable warning.

# Risks
N/A

### CODE REVIEW

File: Adding an Event Listener/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Adding an Event Listener</title>
+  </head>
+  <link rel="stylesheet" href="style.css" />
```
Comment (lycoangelo) [inline Adding an Event Listener/index.html]: `style.css` doesn't exist.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

File: Adding an Event Listener/index.html (hunk 1 above)
Comment (ct-aa) [reply Adding an Event Listener/index.html]: Thanks, @lycoangelo, This is now updated.

File: Adding an Event Listener/app.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+var warning = document.getElementById("warning");
+var button = document.getElementById("makeItRed");
```
Comment (lycoangelo) [inline Adding an Event Listener/app.js]: Please use `const` since the variable values were not reassigned.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

File: Adding an Event Listener/app.js (hunk 2 above)
Comment (ct-aa) [reply Adding an Event Listener/app.js]: Thanks, @lycoangelo, This is now updated.

**Review by lycoangelo — APPROVED** [review]

**Pull request merged**